packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -3,9 +3,115 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
-
 import '../../shared/screen.dart';
 
+const List<String> paths = [
+  '/shoes/..*',
+  '/Clothes/..*',
+  '/Toys/..*',
+  '/Jewelry/..*',
+  '/Watches/..* ',
+  '/Glasses/..*',
+];
+
+List<LinkData> allLinkDatas = [
+  for (var path in paths)
+    LinkData(
+      os: 'Android, iOS',
+      domain: 'm.shopping.com',
+      path: path,
+      domainError: true,
+      pathError: path.contains('shoe'),
+    ),
+  for (var path in paths)
+    LinkData(
+      os: 'iOS',
+      domain: 'm.french.shopping.com',
+      path: path,
+      pathError: path.contains('shoe'),
+    ),
+  for (var path in paths)
+    LinkData(
+      os: 'Android',
+      domain: 'm.chinese.shopping.com',
+      path: path,
+      pathError: path.contains('shoe'),
+    ),
+];
+
+class LinkData {
```

Comment:
consider putting classes related to deep link data models in a different file `deep_links_model.dart` so that `deep_links_screen.dart` can consist of widget classes and UI code only.

Across devtools we generally keep UI classes (flutter widgets), model classes, and controller classes all in their separate categories. Not one class per file by any means, but grouping classes by UI, model, and controller helps with organization